src/parse.rs

```diff
@@ -1580,7 +1721,14 @@ impl ClosureDecorator {
   }
 }
 
-pub fn new_node(kind: Node, span: &Span, args: Vec<Box<Expr>>) -> Box<Expr> {
+pub fn new_node(
+  source_map: &PluginSourceMapProxy,
+  kind: Node,
+  span: &Span,
+  args: Vec<Box<Expr>>,
+) -> Box<Expr> {
+  let line_col = source_map.lookup_char_pos(span.lo());
```

Comment:
Most of the changes in this PR are caused by formatting code after passing the source_map around. 

The meaty part of the change is in this function and `lib.rs` where we acquire the `source_map`.

On this line we map the span's low byte position to a line and column.